main.go

```diff
@@ -224,6 +268,11 @@ func main() {
 	flag.IntVar(&replicationFactor, "replication-factor", 1, "replication factor")
 	flag.DurationVar(&timeout, "timeout", 5*time.Second, "request timeout")
 
+	flag.IntVar(&retryNumber, "retry-number", 0, "number of retries (default 0)")
```

Comment:
0 mean no retries, right ?

Reply:
Right. It is set to be 0 to keep the status quo.

Reply:
yeah, seem the commit/PR description, that explains it